### ISSUE TITLE
The delete_previous playbook fails in some unclear circumstances

### ISSUE THREAD

**Issue by sampaccoud** [openfun/arnold#418]
## Bug Report

**Expected behavior/code**
The `delete_previous` playbook should not fail.

**Actual Behavior**
In some unclear circumstances, the `delete_previous` playbook fails with the following error:
```
TASK [Wait for objects of kind DeploymentConfig to be deleted] ***********************************************************************************************
fatal: [local]: FAILED! => {"msg": "The conditional check '( remaining_objects.resources | length ) == 0' failed. The error was: error while evaluating conditional (( remaining_objects.resources | length ) == 0): 'dict object' has no attribute 'resources'"}
```

**Environment**
- Arnold version: 4.3.1

**Comment by madmatah** [comment]
I managed to reproduce the error by forcing the kubernetes API to fail when the task _Wait for objects of kind {{ kind }} to be deleted_ is executed.

The `delete_previous` playbok should fail on kubernetes API error (on network issue), but with a better error message !
I addressed this particular issue in #435.

**Comment by madmatah** [comment]
Fixed in #435
